src/main/java/tds/exam/results/messaging/ExamCompletedMessageListener.java

```diff
@@ -37,6 +44,10 @@ public void handleMessage(final String examId) {
         LOG.debug("Received completed exam notification for id: {}", examId);
         final TDSReport report = examResultsService.findExamResults(UUID.fromString(examId));
 
-        //TODO ship the report somewhere
+        try {
+            testIntegrationSystemService.sendResults(UUID.fromString(examId), report);
```

Comment:
the audit service also uses the TDSReport object for auditing, so if I make this change, we will be calling examResultsService.findExamResults() twice. I guess that is not a problem, as long as we are caching every call findExamResults().